pkg/frontend/openshiftcluster_putorpatch_test.go

```diff
@@ -138,6 +139,9 @@ func TestPutOrPatchOpenShiftCluster(t *testing.T) {
 							},
 						},
 					},
+					CorrelationData: &api.CorrelationData{
+						StartTime: time.Now(),
+					},
```

Comment:
I'm not sure exactly what to put here to mock?

Reply:
pick a value out of thin air. e.g. `time.Date(2000, 1, 1, 0, 0, 0, 0, time.UTC)`.